pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,22 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	// +optional
+	Config *PrometheusConfiguration `json:"config,omitempty"`
+}
+
+// Config  is the the desired state of Prometheus Configuration
+type PrometheusConfiguration struct {
+	ConfigMapRef *ConfigMapRef `json:"configMapRef,omitempty"`
+
+	CommandOptions []string `json:"command,omitempty" protobuf:"bytes,3,rep,name=commandOptions"`
+}
+
+// ConfigMap Info
+type ConfigMapRef struct {
+	Name      string `json:"name,omitempty"`
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
I think the type of the `Namespace` should be `*string`, and when user only define the Name, we shall use the tidbmonitor's namespace as the configmap's namespace.